src/style.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.container {
+  position: relative;
+  margin-top: 20px;
+}
+
+.box {
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-sizing: border-box;
+  width: 100%;
+  padding: 23px 0 23px 62px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px center;
+  background-size: 19px 19px;
+  font-size: 16px;
+  line-height: 22px;
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+}
+
+.box-sm {
+  padding: 10px 0 11px 33px;
+  background-position: 12px center;
+  background-size: 11px 11px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.box:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.box:focus {
+  border-radius: 3px;
+  box-shadow: 0 0 0 rgba(61, 78, 97, 0.2);
+  background: center / contain no-repeat url("./images/Search.svg"),
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-position: 25px center;
+  background-size: 19px 19px;
```

Comment:
@amitpatiljc there's a side effect - better visible with more contrast gradient:
![Image](https://user-images.githubusercontent.com/39308717/158992021-cc5b13ff-3b84-42ca-a70f-8eaf4a7ddbf9.png)
to avoid 'lining' the gradient - don't state the second measure for `background-size:`
```suggestion
  background-size: 19px;
```